app/src/main/java/com/babylon/wallet/android/utils/DateExtensions.kt

```diff
@@ -55,14 +55,21 @@ fun Instant.dayMonthDateShort(): String {
 fun Instant.dayMonthDateFull(): String {
     val zoneId = ZoneId.systemDefault()
     val currentYear = Instant.now().atZone(zoneId).year
+    val currentDay = Instant.now().atZone(zoneId).dayOfYear
     val instantYear = atZone(zoneId).year
+    val instantDay = atZone(zoneId).dayOfYear
     val format = if (currentYear == instantYear) {
         LAST_USED_DATE_FORMAT_THIS_YEAR
     } else {
         LAST_USED_DATE_FORMAT
     }
+    val prefix = when {
+        currentDay == instantDay -> "Today, "
```

Comment:
⚠️ This will return "Today, " if the day is the same as the previous year. The same will apply below. You need to check if the year is the same too.

Reply:
damn, great catch